src/main/java/mate/jdbc/dao/impl/CarDaoImpl.java

```diff
@@ -0,0 +1,249 @@
+package mate.jdbc.dao.impl;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.dao.CarDao;
+import mate.jdbc.exception.DataProcessingException;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class CarDaoImpl implements CarDao {
+
+    public static final int FIRST_INDEX = 1;
+    public static final int SECOND_INDEX = 2;
+    public static final int THIRD_INDEX = 3;
+
+    @Override
+    public Car create(Car car) {
+        String insertRequest = "INSERT INTO cars (manufacturer_id, model)\n "
```

Comment:
```suggestion
        String insertRequest = "INSERT INTO cars (manufacturer_id, model) "
```
\n is redundant for SQL queries.